Makefile

```diff
@@ -11,7 +11,10 @@ help:
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 bindings: ## generates contract bindings
-	cd contracts && rm -rf bindings/* && ./generate-bindings.sh 
+	cd contracts && rm -rf bindings/* && ./generate-bindings.sh
```

Comment:
think bindings should also generate the eigenpod-bindings, to make sure we don't forget?

Reply:
it does generate https://github.com/Layr-Labs/eigensdk-go/blob/7c76ea7cdb3bc3c0753de009c01054a8c09446d3/contracts/generate-bindings.sh#L59

Reply:
c68b91180062786b36dc2566fb7dc393ac673d89